app/views/shared/_google_analytics.html.haml

```diff
@@ -7,3 +7,28 @@
     gtag('js', new Date());
 
     gtag('config', '#{ENV['GOOGLE_ANALYTICS_ID']}');
+
+  :javascript
+    document.addEventListener("DOMContentLoaded", function() {
```

Comment:
WDYT about listening `turbolinks:load` instead?

It'll allow us to not only track the first-page load but any other page visit.

Reply:
I did not see it enabled in the application layout, I dont think we're using it 🤔

Reply:
OMG

My bad 😛